### DISCUSSION TITLE
Change globe (world) key on mac

### DISCUSSION THREAD

**Discussion by waldemarker** [pqrs-org/KE-complex_modifications#1311] [Q&A]
Hi, the new flat apple magic keyboard is very fine, but I often press accidentally the globe/world button that shows me emojis and symbols. In mac system settings I can disable that. But it would fine to enable that function/button with another rule or key, like Left-Shift+Globe-Button. But I did not find any variable/settings in Karabiner to make this work. Is this possible?

**Comment by gsinti** [comment]
You would need to make it a complex modification.

Something like this should work:

```
{
    "description": "Left Shift + Fn: Globe",
    "manipulators":
    [
        {
            "from":
            {
                "apple_vendor_top_case_key_code": "keyboard_fn",
                "modifiers":
                {
                    "mandatory":
                    [
                        "left_shift"
                    ]
                }
            },
            "to":
            [
                {
                    "apple_vendor_top_case_key_code": "keyboard_fn"
                }
            ],
            "type": "basic"
        }
    ]
}
```

**Comment by waldemarker** [comment]
Hi thank you very much, but how can I import this code in Karabiner on mac?
I can insert only codes on official rule site
